RecoHI/Configuration/python/customise_ECALsequence.py

```diff
@@ -0,0 +1,14 @@
+import FWCore.ParameterSet.Config as cms
+
+def changeHeavyIonsToUseECALGlobalFit(process) :
+    process.load('RecoLocalCalo.EcalRecProducers.ecalGlobalUncalibRecHit_cfi')
```

Comment:
it would be better if you can check first that process has caloReco.
```
if hasattr (process, "caloReco")
... the rest follows
```

this way we can less discrimanately pass this function even to the runTheMatrix.py